src/main/java/ch/njol/skript/lang/SkriptParser.java

```diff
@@ -768,7 +769,7 @@ public final Expression<?> parseExpression(final ExprInfo vi) {
 			
 			final List<int[]> pieces = new ArrayList<>();
 			{
-				final Matcher m = listSplitPattern.matcher(expr);
+				final Matcher m = LIST_SPLIT_PATTERN.matcher(expr);
 				int i = 0, j = 0;
```

Comment:
It might be nice to label the numbers so future people don't get confused when reading this.

Reply:
I agree, but I have plans to add code comments and rename variables and stuff in the entirety of SkriptParser sometime, so I'll leave this for then